File: Kata 8/SimpleMultiplication/SimpleMultiplication.java
/* Kata Level: 8.
 * 
 * This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.
 */

public class SimpleMultiplication {
	
	// Mi solución.
	public static int simpleMultiplication(int n) {
		if (n % 2 == 0)
			return n * 8;
		return n * 9;
	}
	
	// Solución de la comunidad.
	public static int simpleMultiplication1(int n) {
        return n % 2 == 0 ? n * 8 : n * 9;
    }

	public static void main(String[] args) {

		System.out.println(simpleMultiplication(8));
		System.out.println(simpleMultiplication(5));

	}

}

File: Kata 8/SimpleMultiplication/SimpleMultiplication.py
"""
Kata Level: 8.

This kata is about multiplying a given number by eight if it is an even number and by nine otherwise.
"""

# Solución en Python.
	
def simple_multiplication(number) :
    if number % 2 == 0: 
        return number*8 
    else: 
        return number*9

#Solución de la comunidad 1.
	
def simple_multiplication(number) :
    return number * 9 if number % 2 else number * 8

print(simple_multiplication(4))
print(simple_multiplication(5))